docs/http-api.md

```diff
@@ -66,6 +66,47 @@ json and treejson are the same.
 curl -H "X-Org-Id: 12345" "http://localhost:6060/metrics/find?query=statsd.fakesite.counters.session_start.*.count"
 ```
 
+## Find tagged metrics
+
+```
+GET /tags/findSeries
+POST /tags/findSeries
+```
+
+Returns metrics which match tag queries and have received an update since `from`.
+Note: the returned results are not deduplicated and in certain cases it is possible
+that duplicate entries will be returned.
+
+##### Parameters
+
+* expr (required): a list of [tag expressions](#tag-expressions)
+* from: Graphite [from time specification](#fromto) (optional. defaults to now-24hours)
+* format: series-json, lastupdate-json. (defaults to series-json)
+* limit: max number to return, 0 to defer to the instance configured max (default: 0). Note: if limit is 0 or greater than the instance configured max and the result set is greater than the instance configured max, an error is returned. Otherwise, the result set is truncated at the limit
```

Comment:
"instance configured max" sounds too vague. Let's say `http.max-series-per-req` (note, only for these docs, for the cloud docs above it's good as is, as we don't want to expose those details to our cloud users)